src/main/resources/frontend/src/views/AdminView/SubmissionsView.vue

```diff
@@ -20,25 +20,36 @@ import {nameFromNetId} from "@/utils/utils";
 
 const selectedSubmission = ref<Submission | null>(null);
 const selectedStudent = ref<User | null>(null);
+const DEFAULT_SUBMISSIONS_TO_LOAD = 25;
```

Comment:
We need a client side config system similar to #156.

Reply:
I agree, though at this point I'm not sure if it's worth it to set this up in this PR.

Reply:
Agreed. Not worth setting up in this PR, but the reference to the other issue will show up in the history and maybe help us remember where we we wanted to use it when we do implement it.